src/python-bindings/schedd.cpp

```diff
@@ -4403,6 +4404,7 @@ void export_schedd()
         .def("transaction", &Schedd::transaction, transaction_overloads(
             R"C0ND0R(
             Start a transaction with the *condor_schedd*.
+            DEPRECATED. use schedd.Submit instead
```

Comment:
This change is missing from the rendered manual, which I suspect indicates a problem in the build process.

Reply:
```suggestion
            DEPRECATED. use :meth:`schedd.Submit` instead
```
Should create a link to the method in question.